test/bin/manage_build_cache.sh

```diff
@@ -7,7 +7,7 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
-AWS_BUCKET_NAME="${AWS_BUCKET_NAME:-microshift-build-cache}"
+AWS_BUCKET_NAME="microshift-ushift-6071-cache-us-west-2"
```

Comment:
this line change will be removed before merging this PR, add it to speed up the testing of this PR changes